common/logger/Cargo.toml

```diff
@@ -15,3 +15,9 @@ edition = "2018"
 chrono = "0.4.9"
 env_logger = { version = "0.7.1", default-features = false }
 log = "0.4.8"
+serde_json = "1.0.48"
+serde = { version = "1.0.96", features = ["derive"] }
+
+[features]
+names_required = []
```

Comment:
shouldn't this be default and than disabled via no-default otherwise?

Reply:
Again, due to how cargo handles features, this is not as easy as it seems...